splunk_add_on_ucc_framework/templates/oauth.template

```diff
@@ -112,28 +113,31 @@ class {{app_name | lower}}_rh_oauth2_token(admin.MConfigHandler):
     """
 
     def getProxyDetails(self):
-        # Create confmanger object for the app with realm
-        cfm = conf_manager.ConfManager(self.getSessionKey(
-        ), "{{app_name}}", realm="__REST_CREDENTIAL__#{{app_name}}#configs/conf-{{app_name|lower}}_settings")
-        # Get Conf object of apps settings
-        conf = cfm.get_conf('{{app_name|lower}}_settings')
-        # Get proxy stanza from the settings
-        proxy_config = conf.get("proxy", True)
+        try: 
+            proxy_config = conf_manager.get_proxy_dict(logger=logger,
```

Comment:
get_proxy_dict will be added in next solnlib (most likely) but what if someone will be using older version?

Reply:
Added a function `_check_libraries_required_for_oauth` which will raise the exception if latest version of solnlib is not installed, similar to `splunktaucclib` implementation.